### PR TITLE
chore: end to end flow to user create

### PR DESCRIPTION
**Pull request by nicolasburtey**

### CODE REVIEW

File: dev/ory/body.jsonnet [hunk 1]
```diff
@@ -1,7 +1,6 @@
 function(ctx) {
     identity_id: if std.objectHas(ctx, "identity") then ctx.identity.id else null,
     phone: if std.objectHas(ctx.identity.traits, "phone") then ctx.identity.traits.phone else null,
-    transient_payload: if std.objectHas(ctx.flow, "transient_payload") then ctx.flow.transient_payload else null,
```
Comment (nicolasburtey) [inline dev/ory/body.jsonnet:4]: TODO: remove from deployment

File: dev/ory/kratos.yml [hunk 2]
```diff
@@ -170,6 +170,8 @@ identity:
       url: file:///home/ory/phone_email_no_password_v0.identity.schema.json
     - id: phone_or_email_password_v0
       url: file:///home/ory/phone_or_email_password_v0.identity.schema.json
+    - id: username_password_deviceid_v0
+      url: file:///home/ory/username_password_deviceid_v0.identity.schema.json
```
Comment (nicolasburtey) [inline dev/ory/kratos.yml:174]: TODO: add to deployment

File: dev/ory/oathkeeper_rules.yaml [hunk 3]
```diff
@@ -26,11 +26,11 @@
       config:
         claims: '{"sub": "{{ print .Subject }}"}'
 
-- id: galoy-backend
+- id: device-login-routes
```
Comment (nicolasburtey) [inline dev/ory/oathkeeper_rules.yaml:29]: TODO: update to deployments

File: src/servers/middlewares/auth-router.ts [hunk 4]
```diff
@@ -199,4 +200,55 @@ authRouter.get("/clearCookies", async (req, res) => {
...
+      deviceId,
+    })
+
+    if (authToken instanceof Error) {
+      // TODO open telemetry
+      return res.status(500).send({ error: authToken.message })
+    }
+
+    return res.status(200).send({
+      result: authToken,
```
Comment (UncleSamtoshi) [inline src/servers/middlewares/auth-router.ts:236]: should this value be changed to just be `{ authToken }` which is what we call it in other places?

File: dev/ory/body.jsonnet (hunk 1 above)
Comment (ntheile) [reply dev/ory/body.jsonnet:4]: done

File: dev/ory/kratos.yml (hunk 2 above)
Comment (ntheile) [reply dev/ory/kratos.yml:174]: done

File: dev/ory/oathkeeper_rules.yaml (hunk 3 above)
Comment (ntheile) [reply dev/ory/oathkeeper_rules.yaml:29]: done

**Comment by ntheile** [comment]
need to run `make e2e-codegen` to update generated for tests

**Review by ntheile — DISMISSED** [review]

**Review by ntheile — APPROVED** [review]

**Pull request merged**